ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/AffectedClustersTable.tsx

```diff
@@ -14,7 +14,7 @@ import { displayClusterType } from '../utils/stringUtils';
 
 export const sortFields = {
     Cluster: 'Cluster',
-    ClusterType: 'Cluster Type',
+    ClusterType: 'Cluster Platform Type',
```

Comment:
Just discovered this was [not the correct field](https://github.com/stackrox/stackrox/pull/10844/files#diff-4e9e719f0bb6eaeccc0a709142b9a556570bccf8b869de2d33da8147bb052444R146), so I'm piggybacking this change on a related sorting PR...